tests/wdio/reports/result-orion-fusion-theme-chrome-terra-dev-site.json

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
We prob need to add reports to the gitIgnore

Reply:
Added here https://github.com/cerner/terra-dev-site/pull/291/commits/fc27778c1e6288dec7f38340407ded016fe10eb3